content/flutter-publishing/publishing-to-app-store.md

```diff
@@ -66,21 +66,30 @@ Once the Apple Developer Portal has been enabled for the account or team the app
 2. Click **App Store Connect**.
 3. If you have several keys available, select the right key in the **App Store Connect API key** field.
 4. Mark the **Publish even if tests fail** checkbox to continue uploading the app artifact even when the tests failed.
-4. Mark the **Submit to TestFlight beta review** checkbox to submit the build for beta review and prepare it for distributing to beta testers. Note: This action is performed during [post-processing](#post-processing-of-app-store-connect-distribution).
-5. Mark the **Distribute to beta groups** checkbox and enter the names of the beta groups to automatically distribute the build to the testers in those groups once the build has passed beta review. Note: This action is performed during [post-processing](#post-processing-of-app-store-connect-distribution).
-6. Select **Enable App Store Connect publishing** at the top of the section to enable publishing.
+5. Select **Enable App Store Connect publishing** at the top of the section to enable publishing.
 
-Once you have successfully set up publishing to App Store Connect, Codemagic will automatically distribute the app to App Store Connect every time you build the workflow. Note that you must manually submit the app to App Store in App Store Connect.
+To continue, either configure the submission to TestFlight or App Store, or choose both options.
 
-## Submitting an app to App Store
+#### Submitting an app to TestFlight
 
-To make your iOS app available to the public, it must be submitted for review in App Store Connect.
+1. Mark the **Submit to TestFlight beta review** checkbox to submit the build for beta review and prepare it for distributing to beta testers. Note: This action is performed during [post-processing](#post-processing-of-app-store-connect-distribution).
+2. Mark the **Distribute to beta groups** checkbox and enter the names of the beta groups to automatically distribute the build to the testers in those groups once the build has passed beta review. Note: This action is performed during [post-processing](#post-processing-of-app-store-connect-distribution).
+
+Once you have successfully set up publishing to App Store Connect, Codemagic will automatically distribute the app to App Store Connect every time you build the workflow. The application has to be manually submitted to App Store.
```

Comment:
I would omit this paragraph here as it's irrelevant. The point is that without selecting any of the options (TestFlight or App Store review), the build would be still published to TestFlight but just not submitted to review.